sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -545,6 +545,13 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val STREAMING_POLLING_DELAY =
+    SQLConfigBuilder("spark.sql.streaming.pollingDelay")
```

Comment:
I changed this one to `timeConf` so that people can use a time string such as `10 seconds`. Then we don't need to add `Ms` suffix to the configuration key.